string[] topic   # container for message topics
string[] value   # values for the messages
string[] raw_data # stores last raw data value
int64[] msg_count # stores total count for all messages uptill now
